doc/terminology.rst

```diff
@@ -27,15 +27,15 @@ Terminology
 
 ----
 
-**Coordinate:** An array that labels a dimension of another ``DataArray``. Loosely, the coordinate array's values can be thought of as tick labels along a dimension. There are two types of coordinate arrays: *dimension coordinates* and *non-dimension coordinates* (see below). A coordinate named ``x`` can be retrieved from ``arr.coords[x]``. A ``DataArray`` can have more coordinates than dimensions because a single dimension can be assigned multiple coordinate arrays. However, only one coordinate array can be a assigned as a particular dimension's dimension coordinate array. As a consequence, ``len(arr.dims) <= len(arr.coords)`` in general.
+**Coordinate:** An array that labels a dimension or set of dimensions of another ``DataArray``. In the one-dimensional case, the coordinate array's values can loosely be thought of as tick labels along a dimension, whereas :doc:`multidimensional coordinates are often used when the data's physical coordinates differ from their logical coordinates <examples/multidimensional-coords>`. There are two types of coordinate arrays: *dimension coordinates* and *non-dimension coordinates* (see below). A coordinate named ``x`` can be retrieved from ``arr.coords[x]``. A ``DataArray`` can have more coordinates than dimensions because a single dimension can be assigned multiple coordinate arrays. However, only one coordinate array can be a assigned as a particular dimension's dimension coordinate array. As a consequence, ``len(arr.dims) <= len(arr.coords)`` in general.
```

Comment:
> are often used when the data's physical coordinates differ from their logical coordinates

What does this mean? Is it climate-focused? (I'm not sure I have something better in mind yet, though)

Reply:
This was taken from https://xarray.pydata.org/en/latest/examples/multidimensional-coords.html, and it was the best I could think of for a concise explanation. I interpreted it to refer to physical coordinates (like latitude and longitude) that do not always line up with the axes ("logical coordinates") of one's data (which is often the case when working with Earth-based data that are on some grid other than latitude/longitude). I'd be glad to change it to something else though if there are any suggestions!

Reply:
That makes sense. I think it's probably too climate focused atm. I think we could either add `for example, in climate datasets...`, or remove that clause, or if anyone has suggestions?

Reply:
I think `in geoscience datasets` may be more appropriate/general than `in climate datasets` (speaking as a meteorologist :grin:), but I can easily make that change if no other suggestions come up.

Reply:
Since no other suggestions came in, I made this update. It also seemed to work better moving this to the **Non-dimension coordinate** definition (if I should move it back, just let me know).